utils/python/CIME/preview_namelists.py

```diff
@@ -76,12 +83,14 @@ def create_namelists(case):
             if 'python' in first_line:
                 expect(False, detail)
             else:
-                run_cmd_no_fail("%s %s" % (cmd, caseroot), verbose=True)
+                run_cmd_no_fail("%s %s" % (cmd, caseroot), verbose=False)
```

Comment:
I would like to defer to a later PR and get this PR accepted as soon as possible due to the resolution of issue #916.